### ISSUE TITLE
[NPC] Elder Diemetradon

### ISSUE THREAD

**Issue by Mewmewww** [Atlantiss/NetherwingBugtracker#1722]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)

**Description**: 
Seems to attack 3 times in a row in some cases.
**Current behaviour**:
I haven't seen them use 3 attacks in a row before. https://imgur.com/a/b5o6uLB
**Expected behaviour**: 
I don't believe they should attack me 3 times in a row. no signs of it on wowhead, wowwiki etc.
**Server Revision**: 
2186

**Comment by Firroth** [comment]
Well nothing wrong with it since this mob uses - [thrash](http://classicdb.ch/?spell=3417), which he always had.

**Comment by Mewmewww** [comment]
good catch. i thought that classicdb was made on the basis of a private server. Let me to believe that, that wasn't good enough evidence
